Add new package "IPython Pick"

marciomazza opened the pull request.

FichteFoll commented:
You must move your test files to a subdirectory. Otherwise, ST will load them automatically and override `sys.modules["sublime_"]`, which will result in numerous failures for other plugins.

[Semantic versions](http://semver.org) consist of exactly three numeric parts.

marciomazza commented:
Thanks for the revision.

I moved the files to the subdirectory "tests" and made some more provisioning on the sublime modules mocking.
Also changed the tag to 0.1.0
